pages/plugins.rst

```diff
@@ -77,12 +77,12 @@ Use it like this::
 
 This will create a checkout of the meta project in your current work dir. Now change to the ``graylog-project`` directory and execute the step which to download the necessary base modules::
 
-  $ scripts/bootstrap
+  $ graylog-project bootstrap
```

Comment:
In the graylog-projects readme, the bootstrap command is followed by the repo url `github://Graylog2/graylog-project.git. Should we add this here as well?